config/routes.rb

```diff
@@ -219,6 +219,10 @@
 
       resources :statements, only: %i[index show], param: :ecf_id
     end
+
+    namespace :teacher_record_service, path: "teacher-record-service", defaults: { format: :json } do
```

Comment:
this seems to be under v3 although the controller is v1, I suggest this is decoupled from the lead provider API as we also have feature flags hiding the API currently. and possibly move this to a v1 as they shouldn't be connected or block each other esp regarding versions?